hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/BackendEntityResolverTest.java

```diff
@@ -567,6 +615,54 @@ public void checkBackendEntityGeneratedFromInstrumentedMongoEvent() {
             .getString(), "62646630336466616266356337306638");
   }
 
+  @Test
+  public void checkBackendEntityGeneratedFromInstrumentedMongoEventOtelFormat() {
+    Event e = Event.newBuilder().setCustomerId("__default")
+        .setEventId(ByteBuffer.wrap("bdf03dfabf5c70f8".getBytes()))
+        .setEntityIdList(Arrays.asList("4bfca8f7-4974-36a4-9385-dd76bf5c8824")).setEnrichedAttributes(
+            Attributes.newBuilder().setAttributeMap(
+                Map.of("SPAN_TYPE", AttributeValue.newBuilder().setValue("EXIT").build())).build())
+        .setAttributes(Attributes.newBuilder().setAttributeMap(Map
+            .of( OTelDbAttributes.DB_SYSTEM.getValue(), buildAttributeValue(OTelDbAttributes.MONGODB_DB_SYSTEM_VALUE.getValue()),
+                OTelDbAttributes.NET_PEER_NAME.getValue(), buildAttributeValue("mongodb0"),
+                OTelDbAttributes.MONGODB_COLLECTION.getValue(), buildAttributeValue("sampleshop.userReview"),
+                "span.kind", buildAttributeValue("client"),
+                OTelDbAttributes.DB_OPERATION.getValue(), buildAttributeValue("FindOperation"),
+                OTelDbAttributes.NET_PEER_PORT.getValue(), buildAttributeValue("27017"),
```

Comment:
Do you think, we need one more test case where if the connection string is present, it will get backendURI from it?

Reply:
Those cases have already been covered in utility test. So I think this is mostly customary.